Policies/github-terms-of-service.md

```diff
@@ -36,95 +36,118 @@ Thank you for using GitHub! We're happy you're here. Please read this Terms of S
+7. “The User,” “You,” and “Your” refer to the individual person, company, or organization that has visited or is using the Website or Service; that accesses or uses any part of the Account; or that directs the use of the Account in the performance of its functions. 
+8. The “Website” refers to GitHub’s website located at [github.com](https://github.com/), and all content, services, and products provided by GitHub at or through the Website. It also refers to GitHub-owned subdomains of github.com, such as [education.github.com](https://education.github.com/) and [pages.github.com](https://pages.github.com/).
 
 ### B. Account Terms
 **Short version:** *User Accounts and Organizations have different administrative controls; a human must create your Account; you must be 13 or over; you must provide a valid email address; and you may not have more than one free Account. You alone are responsible for your Account and anything that happens while you are signed in to or using your Account. You are responsible for keeping your Account secure.*
 
 #### 1. Account Controls
+You only need to read the second bullet point below if you are dealing with an Organization.
 - Users. Subject to these Terms, you retain ultimate administrative control over your User Account and the Content within it.
-
 - Organizations. The "owner" of an Organization that was created under these Terms has ultimate administrative control over that Organization and the Content within it. Within the Service, an owner can manage User access to the Organization’s data and projects. An Organization may have multiple owners, but there must be at least one User Account designated as an owner of an Organization. If you are the owner of an Organization under these Terms, we consider you responsible for the actions that are performed on or through that Organization.
 
 #### 2. Required Information
-You must provide a valid email address in order to complete the signup process. Any other information requested, such as your real name, is optional, unless you are accepting these terms on behalf of a legal entity (in which case we need more information about the legal entity) or if you opt for a [paid Account](#k-payment), in which case additional information will be necessary for billing purposes.
+The following information is required:
+- A valid email address to complete the signup process. 
+- If you represent a legal entity, certain information about the legal entity.
+- For [paid Accounts](#k-payment), certain information for billing purposes.
+
+Other information requested (which could include your real name) is optional.
```

Comment:
`All other information`?

Reply:
Yes, could be better with 'all'. However, it isn't really needed, and one school of thought might say that fewer words in this case is better. Either way, I don't think it matters much.

Reply:
>Either way, I don't think it matters much.

The claimed goal of your PR is to make the text shorter.

Reply:
I don't think I made that the claimed goal of the PR. The rule of thumb was a general one. I wanted to say that it was a general principle that mostly applied (but not necessarily always).

If you read my PR, you'll see that I do write that one of my changes was to split one sentence into many sentences (which ultimately actually meant more words.)